scrooge-generator/src/main/resources/scalagen/finagleService.scala

```diff
@@ -119,6 +120,7 @@ class {{ServiceName}}$FinagleService(
   // ---- end boilerplate.
 
 {{/hasParent}}
+  private[this] val scopedStats = if (serviceName != "") stats.scope(serviceName) else stats
```

Comment:
is it valuable to scope these stats with a name?  it seems unlikely that a single service will expose more than one thrift server.